src/index.ts

```diff
@@ -148,18 +154,19 @@ class PgAnonymizer extends Command {
                   }, extension)(v, table);
                 }
                 return replacement;
+              } else {
+                if (cols[k] === "email") return faker.internet.email();
+                if (cols[k] === "name") return faker.name.findName();
+                if (cols[k] === "description") return faker.random.words(3);
+                if (cols[k] === "address") return faker.address.streetAddress();
+                if (cols[k] === "city") return faker.address.city();
+                if (cols[k] === "country") return faker.address.country();
+                if (cols[k] === "phone") return faker.phone.phoneNumber();
+                if (cols[k] === "comment") return faker.random.words(3);
+                if (cols[k] === "birthdate")
+                  return postgreSQLDate(faker.date.past());
+                return faker.random.word();
```

Comment:
I originally put these lines out of else based on the "avoid else" statement I read here and there (it's about cyclomatic complexity). Not sure it's a piece of good advice though, so feel free to keep it like this!